lib/dice/src/lib.rs

```diff
@@ -238,3 +242,38 @@ impl AliasOkm {
         Self(okm_from_seed_no_extract(cdi, "attestation".as_bytes()))
     }
 }
+
+/// SwdspOkm is a type that represents the output keying material (OKM) used
+/// to create the Swdsp key. This key is used as an embedded CA by the task
+/// that owns the SWD interface to the service processor.
```

Comment:
Suggest changing this and all structure names to refer to 'measurement' vs 'swd' since SWD is just the underlying protocol and I spent some time trying to figure out why SWD needed a certificate (SWD itself doesn't need it but we want it for measuring!)

Reply:
Reasonable. Might be best to use the same name as the task as the prefix (in UpperCamelCase) : SpMeasureOkm etc.

Reply:
should be resolved